### PR TITLE
Add ForgotPassword component

### PR DESCRIPTION
**Pull request by VitaliKozhurov**

### CODE REVIEW

**Review by DenisKulik — APPROVED** [review]
good!

File: src/components/auth/forgotPassword/ForgotPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { ControlledInput } from '../../controlled'
+
+import s from './ForgotPassword.module.scss'
+
+import { TypographyVariant } from '@/common'
+import { Button, Card, Typography } from '@/components'
+import { UseForgotPassword } from '@components/auth/forgotPassword/UseForgotPassword.ts'
```
[the remark this answers is not in the document]
Comment (VitaliKozhurov) [reply src/components/auth/forgotPassword/ForgotPassword.tsx]: Fix imports!

File: src/components/auth/forgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+  return (
+    <Card className={s.formWrapper}>
+      <form className={s.form} onSubmit={onSubmit}>
+        <Typography className={s.formHeader} as="h2" variant={TypographyVariant.Large}>
+          Forgot your password?
+        </Typography>
+        <ControlledInput
+          className={s.input}
+          name={'email'}
```
[the remark this answers is not in the document]
Comment (VitaliKozhurov) [reply src/components/auth/forgotPassword/ForgotPassword.tsx]: Fix brackets for text attributes!

**Review by vitalyabedik — APPROVED** [review]
Good!

**Pull request merged**